packages/react-wildcat-handoff/src/utils/defaultTemplate.js

```diff
@@ -10,6 +11,7 @@ module.exports = function defaultTemplate(cfg) {
 
     const wildcatConfig = cfg.wildcatConfig;
     const clientSettings = wildcatConfig.clientSettings;
+    const protocol = (((wildcatConfig || {}).serverSettings || {}).appServer || {}).protocol;
```

Comment:
Unit tests fail because some of the stubs do not have appServer. I figured it'd make more sense to add the check here then to change the tests, so that way wildcat won't error out if someone deletes the protocol. I can change the tests though if you think that makes more sense